_includes/operator/install.md

```diff
@@ -0,0 +1,78 @@
+## Install StorageOS operator
+
+The StorageOS cluster operator is the recommended installation procedure to
+deploy a StorageOS cluster.
+
+The StorageOS cluster operator is a [Kubernetes native
+application](https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/)
+developed to deploy and configure StorageOS clusters, and assist with
+maintenance operations.
+
+The StorageOS operator can be installed with Helm.
+
+### Install
+
+```bash
+helm repo add storageos https://charts.storageos.com
+helm install storageos/storageoscluster-operator --namespace storageos-operator
+```
+
+> The Helm chart can be found in the [Charts public
+> repository](https://github.com/storageos/charts).
+
+> The StorageOS Cluster Operator source code can be found in the
+> [cluster-operator repository](https://github.com/storageos/cluster-operator).
+
+The operator is a Pod manged by a Deployment Controller. Once the Pod is ready,
```

Comment:
The operator is a Pod managed

Reply:
I think we should not mention pod, but refer as a kubernetes controller that watches `StorageOSCluster` CRD.

Reply:
fixed the typo and I applied a better definition for the operator. Mention it as CRD. 

Thank you